esp/esp/program/modules/handlers/resourcemodule.py

```diff
@@ -674,6 +678,79 @@ def resources(self, request, tl, one, two, module, extra, prog):
         #   Display default form
         return render_to_response(self.baseDir()+'resource_main.html', request, context)
 
+    @aux_call
+    @needs_admin
+    def newassignment(self, request, tl, one, two, module, extra, prog):
```

Comment:
This is an oddball but I have a class with two teachers. If I assign 1 resource to the class and refresh the page, suddenly the resource shows up twice for that class? But if I go to the next block it only shows up as one resource again. :/ Not entirely sure it has to do with the two teachers thing, but it doesnt happen for my single teacher class so that's my only guess why the resource fake-doubles.

Reply:
That is...very strange...I've got classes with multiple teachers, and I haven't experienced this problem in my testing. Also, the number of teachers should not be a factor when the resources are assigned. Is it definitely only one resource that is assigned (in the admin panel)?